app/proto/search.proto

```diff
@@ -40,7 +40,7 @@ message SearchReq {
   // * `or` to search for one or another search term
   // * `-` (minus) to ignore a search term
   // in the fuzzy search these have no meaning
-  string query = 1;
+  string statement = 14;
```

Comment:
I don't think statement is a good name. Also I think we should change the protocol separately from internal changes in the backend.

Reply:
"statement" is what SQLA generally uses. SQL_statement?

Reply:
There's no sql in our protocol between frontend and backend.

Reply:
How about `search_string` or `search_term`?